assets/js/breadcrumbs/mobile-breadcrumb.js

```diff
@@ -0,0 +1,132 @@
+(function() {
```

Comment:
Thanks for the reminder about IIFE for global var prevention. It'd been awhile since I used that pattern.